test-projects/simple-database/project.visivo.yml

```diff
@@ -158,7 +158,7 @@ traces:
             - props.y
   - name: 3d Line Two
     model: ref(test table)
-    cohort_on: "'Line Two'"
+    #cohort_on: "'Line Two'"
```

Comment:
Can we get rid of the comments?

Reply:
Ah yeah that would be good. I'll get those removed in the next pr 👍